module/stylesheets/components/_card.scss

```diff
@@ -0,0 +1,10 @@
+// Card
+
+.card{
+  padding: $card-spacer-y $card-spacer-x;
+  min-height: $card-min-height;
+}
+.card-raised{
```

Comment:
`.car-raised` should be in `.card` class as is should be as `.card .card-raised`